index.html

```diff
@@ -12,55 +13,74 @@
+      </div>
+    </header>
+    <div class="CTA">
+      <img src="/img/fun-bus.jpg" class="CTA-image" alt="Fun Bus">
+      <h2>Welcome To Fun Bus!</h2>
+      <p> Traveling expedition modern, clean webdesign blogger clean website theme website modern. Design pretty design, travelblogger adventure WordPress wanderlust theme blogger website expedition theme travelblogger. Adventure fun traveler pretty design website expedition.</p>
+    </div>
     <section class="content-section">
       <div class="text-content">
         <h2>Let's Go!</h2>
-        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty simple traveling fun WordPress wanderlust darn simple organized.</p> 
-        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p>
+        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure.
+          Pretty simple traveling fun WordPress wanderlust darn simple organized.</p>
+        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized
+          website theme.</p>
       </div>
       <div class="img-content">
         <img src="img/adventure.jpg" alt="Let's go on an adventure!">
       </div>
     </section>
     <section class="content-section inverse-content">
       <div class="img-content">
-          <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
+        <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
       </div>
       <div class="text-content">
         <h2>Adventure Awaits</h2>
-        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty simple traveling fun WordPress wanderlust darn simple organized.</p> 
-        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p>
+        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure.
+          Pretty simple traveling fun WordPress wanderlust darn simple organized.</p>
+        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized
+          website theme.</p>
       </div>
-     
+
     </section>
     <section class="content-destination">
       <h2>Pick Your Destination</h2>
-      <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
+      <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign.
+        Traveler blogger website design expedition clean excursion traveling.</p>
       <img src="img/destination.jpg" alt="Second slide">
     </section>
     <section class="content-pick">
 
```

Comment:
Please try and be consistent with spacing out sections/divs/ect so its easier to read and digest code. Awesome job on the HTML